modules/contacts/site.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+function contact_phone_verify(){
+    var contact_phone=document.getElementById("contact_phone");
+    contact_phone.value=contact_phone.value.replace(/\D/g,'');
```

Comment:
Please no. People often write spaces (even UTF-8 ones, not just ASCII space character), `+`, `#`, `*`, "short pause", "long pause" (both of these represented by quite different characters on different phones...), etc. - needed for conference calls, special systems, operator messages, etc. Even overall length doesn't make much sense because many people use fixed **one-digit numbers** for certain important contacts (e.g. inside a small company).

So, I for myself would be very afraid to impose basically any restrictions here without knowing the context better. And Cypht does **not** have any context IMHO, so rather no restrictions :wink:.

Just my 2 cents.

Reply:
Also, there are numbers like 1 877 Wiki WEB